### PR TITLE
Rename griffon to assurance

### PR DESCRIPTION
**Pull request by PravinPK**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
   "repository": {
     "type": "git",
-    "url": "https://github.com/adobe/cordova-acpgriffon"
+    "url": "https://github.com/adobe/cordova-aepassurance"
   },
   "keywords": [
     "adobe",
+    "aepassurance",
+    "assurance",
     "griffon",
```
Comment (ravjain-adb) [inline package.json:20]: Don't want to remove this griffon ?

File: plugin.xml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+        version="0.0.1"
         xmlns="http://apache.org/cordova/ns/plugins/1.0"
         xmlns:android="http://schemas.android.com/apk/res/android">
-    <name>cordova-acpgriffon</name>
-    <description>ACPGriffon Plugin for Cordova</description>
+    <name>cordova-aepassurance</name>
+    <description>AEPAssurance Plugin for Cordova</description>
     <license>Apache 2.0</license>
-    <keywords>cordova,adobe,griffon,acpgriffon,marketing</keywords>
+    <keywords>cordova,adobe,assurance,aepassurance,griffon,acpgriffon,marketing</keywords>
```
Comment (ravjain-adb) [inline plugin.xml:9]: same here don't want to remove griffon

**Review by ravjain-adb — APPROVED** [review]
minor comment otherwise LGTM

File: package.json (hunk 1 above)
Comment (PravinPK) [reply package.json:20]: The product name is still Griffon,
So it could be a hint when people search with that name

**Pull request merged**